cmd/edit/ingress/cmd.go

```diff
@@ -63,6 +65,20 @@ var Cmd = &cobra.Command{
 	},
 }
 
+func areLocalFlagsUnchanged(cmd *cobra.Command) bool {
+	parentFlagSet := cmd.Parent().Flags()
+	localFlagsExcluded := []string{"cluster"}
```

Comment:
Tried to make sure that if we add more local flags to this command this would still work. Open to suggestions

Reply:
I am not sure the logic is correct in general.
We would like to go to interactive if the minimal set of flags wasn't specified for this command.
I think using a generic function would be difficult to implement.
Take a look at what we do on other commands, where we don't enable interactive mode globaly but ask for it per feature set. for example create/machine_pool/cmd.go all the replicas and autoscaling flags

Reply:
Oh no, this is not general in the sense that any command should use it.
It is just for this one atm, given that cluster has it's own check and is needed to know which cluster the ingress id is from anyway.
The other parameters are 'optional' in the sense that they create some sort of change when specified. So for the current moment we can use such an approach, but maybe at some point we'll have to change the contents of this function